simulator/modules/fetch/bpu/t/unit_test.cpp

```diff
@@ -42,6 +42,18 @@ TEST_CASE( "Static, all branches taken")
     CHECK( bp->get_target(PC) == target);
 }
 
+TEST_CASE( "Backward only branches taken")
```

Comment:
Could we also test that forward branches are predicted as not taken?

Reply:
Yes, sure, I will add this test